src/pages/home/_hero.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled, { keyframes } from 'styled-components'
+import PropTypes from 'prop-types'
+import VerticalCarousel from './_vertical-carousel'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+import { Container, Box, Flex, Show } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { Localize, localize } from 'components/localization'
```

Comment:
I think this file is no longer there from the upstream.

Reply:
Deleted.